src/libraries/System.Private.CoreLib/src/README.md

```diff
@@ -9,12 +9,17 @@ The goal is to have the majority of code located in this folder, as that code is
 Any runtime-specific `partial` part which also has a shared part should use a runtime-specific file name suffix to ease the navigation.
```

Comment:
There are other CoreCLR and Mono mentions in this file you might want to add NativeAOT to?